app.py

```diff
@@ -197,14 +211,24 @@ def _update_rust_deps_if_required(source, new_commit):
         console.print(f"rust :crab: libraries remain the same ({old_versions}).")
```

Comment:
LOL… I know this isn't part of your commit, but I was just curious if you know: Where is `:crab:` being translated into the :crab: emoji that I see in your console output?

Reply:
XD

rich ftw

Reply:
whoops, i meant `python -m rich.emoji`

In my experience, it seems to be using the same names used on github flavored markdown

Reply:
Nice! _Of course_ it's rich. I ran into this just a few weeks ago and totally forgot!